substrate-archive-common/src/util.rs

```diff
@@ -64,10 +31,13 @@ pub fn init_logger(std: log::LevelFilter, file: log::LevelFilter) {
         .debug(Color::Blue)
         .trace(Color::Magenta);
 
-    // let mut log_dir = dirs::data_local_dir().expect("failed to find local data dir for logs");
-    // log_dir.push("substrate_archive");
-    let mut log_dir = substrate_dir();
-    create_dir(log_dir.as_path());
+    let mut log_dir = match substrate_dir() {
+        Err(e) => panic!(e),
+        Ok(dir) => dir,
+    };
+    if create_dir(log_dir.as_path()).is_err() {
+        panic!("A directory in the path of the substrate directory could not be created")
+    }
```

Comment:
I see what you're doing here, and I think it's better than before. Ideally I think we should hand the error ever upwards, i.e. make `init_logger()` return `Result` and so forth all the way up to the main binary. That's where we want the panics to happen.

Reply:
#122